index.js

```diff
@@ -18,21 +18,21 @@ Pretty print a size from bytes
 */
 
 module.exports = function(size, nospace, one, places) {
-    var mysize, f;
+    let mysize, f;
     places = places || 1;
 
-    sizes.forEach(function(f, id) {
+    sizes.forEach((unit, id) => {
         if (one) {
-            f = f.slice(0, 1);
+            unit = unit.slice(0, 1);
         }
-        var s = Math.pow(1024, id),
-            fixed;
+        const s = Math.pow(1024, id);
+        let fixed;
```

Comment:
For code clarity, mine also changing lines 42 & 43 to use `unit` instead of `f`?